main/BUILD

```diff
@@ -0,0 +1,15 @@
+load("@rules_python//python:defs.bzl", "py_binary")
```

Comment:
generally projects only have one build file at the top level directory

Reply:
https://github.com/bazelbuild/rules_python/issues/325
The example here shows that I need a BUILD in each directory (src and main). I also tried to use one BUILD file at the top level and refer to files with relative path, but it told me to " Add a BUILD file to a directory to mark it as a package.
 - /usr/local/google/home/hsqq/toxicity/sophie/toxicity-detector/main and referenced by '//:train_classifier_g'"

Reply:
Ok! I stand corrected :)